python/lsst/dax/imgserv/locateImage.py

```diff
@@ -189,12 +189,17 @@ def getImageByDataId(self, ra, dec, width, height, qResults, cutoutType="arcseco
         xyWcs = wcs.skyToPixel(raDec)
         x0 = img.getX0()
         y0 = img.getY0()
-        xyCenter = afwGeom.Point2I(int(xyWcs.getX() - x0), int(xyWcs.getY() - y0))
+        xyCenter_x0 = xyWcs.getX() - x0
+        xyCenter_y0 = xyWcs.getY() - y0
+        # specify the center for the cutout
+        pixULX = int(xyCenter_x0 - width/2.0)
+        pixULY = int(xyCenter_y0 - height/2.0)
+        xyCenter = afwGeom.Point2I(pixULX, pixULY)
         if cutoutType == 'pixel':
-            imgSub = _cutoutBoxPixels(img, xyCenter, width, height, self._log)
+            imgSub = _cutoutBoxPixels(img, xyCenter, width, height, wcs, self._log)
+            self._log.info("ra=%f dec=%f xyWcs=(%f,%f) x0y0=(%f,%f) xyCenter=(%f,%f)", ra, 
+                    dec, xyWcs.getX(), xyWcs.getY(), x0, y0, xyCenter.getX(), xyCenter.getY())
```

Comment:
Moved the log stmt ahead of the _cutoutBoxPixels() call for usefulness, but should stay inside the if stmt since the else case would circle back to if "pixel" block, so will be redundant to have it outside.